app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -308,64 +339,39 @@ private void restoreExistingSelection(boolean isOrientationChange) {
         }
     }
 
-    private void setupToolbar(AdapterView view) {

```

Comment:
After staring at this code for a little while, I finally realized that this was only relevant for pre-honeycomb devices, because on any device where the call to `tryToAddSearchActionToAppBar` in `onCreateOptionsMenu` is successful, the toolbar created by this method gets hidden. I wanted to make this clearer in the code, so renamed this to `setupPreHoneycombFooter`, and moved all of the code that wasn't strictly relevant to that to other places (mainly `setupUI`)